content/learning-paths/cross-platform/eigen-linear-algebra-on-arm/eigen-intro.md

```diff
@@ -97,25 +98,25 @@ typedef Matrix<int, 1, 4> RowVector4i;
 typedef Matrix<std::complex<float>, 1, 4> RowVector4cf;
 ```
 
-All basic mathematical operations and major known functions are defined on top of those classes.
+All basic mathematical operations and major known functions are defined on top of these classes.
 
 There are more specialized classes for both storage types, like the `Tensor` class and operations, like FFT and Solvers, in the `unsupported` folder.
 
-In this Learning Path you are are going to test Eigen with a few examples with emphasis on Arm CPUs. For full documentation and a list of all supported types and operations on them, you should read the [Eigen documentation](https://libeigen.gitlab.io/docs/).
+In this Learning Path, you will test Eigen using a few examples with emphasis on Arm CPUs. For full documentation and a list of all supported types and operations on them, see the [Eigen documentation](https://libeigen.gitlab.io/docs/).
 
 ## Numerical Solvers
 
-Apart from basic vector and matrix types, Eigen also provides built-in methods for numerical solving on those matrices.
+Besides basic vector and matrix types, Eigen also provides built-in methods for numerical solving on the matrices.
 
-This Learning Path does not go into depth as to what exactly the solvers do, but rest assured that numerical solvers are actually very useful not only for a standpoint of Mathematics, but also actual applications that you are probably using in one way or the other, possibly without knowing it. 
+This Learning Path does not go into depth on what the solvers do, but the numerical solvers are useful not only as a standpoint of Mathematics, but also as applications that you are likely using in one way or the other, possibly without knowing. 
```

Comment:
I think the phrase should be "from the standpoint of Mathematics", meaning that they not only useful in Pure Mathematics, but have applications elsewhere. I might be mistaken but according to Cambridge dictionary, "from the standpoint of" is the correct expression.